CONTRIBUTING.md

```diff
@@ -30,6 +30,14 @@ duplication of work, and help you to craft the change so that it is successfully
 project.
 * **Small Changes** can be crafted and submitted to [GitHub Repository][github] as a Pull Request.
 
+## <a name="projects"></a> Projects
+
+Projects are either active, unmaintained or deprecated.
+
+* "active" means that the project is under development and that releases are produced once a year. All projects that are not unmaintained or deprecated are active.
```

Comment:
I tried to apply my suggeested change on top of this but It seems that I don't have permission to write to this repo.